doc/classes/SpringArm.xml

```diff
@@ -33,17 +41,25 @@
 			<argument index="0" name="RID" type="RID">
 			</argument>
 			<description>
+			Remove the given objects from the objects excluded from the collision check
```

Comment:
Indentation.

Reply:
Should also be third person and lacks dot at the end. The argument should be a single object I guess? So the description should match that, e.g. "Removes the given object from the list of objects excluded from the collision check."